assets/xml/objects/object_ma1.xml

```diff
@@ -1,96 +1,116 @@
-        <Animation Name="object_ma1_Anim_015B7C" Offset="0x15B7C" />
-        <Animation Name="object_ma1_Anim_016588" Offset="0x16588" />
-        <Animation Name="object_ma1_Anim_0180DC" Offset="0x180DC" />
-        <Animation Name="object_ma1_Anim_018948" Offset="0x18948" />
-        <Animation Name="object_ma1_Anim_01B76C" Offset="0x1B76C" />
+        <!-- Romani's Bow DisplayList and Textures -->
+        <DList Name="gRomaniBowDL" Offset="0x3B0" />
+        <Texture Name="gRomaniBowTLUT" OutName="romani_bow_tlut" Format="rgba16" Width="16" Height="16" Offset="0x5E0" />
+        <Texture Name="gRomaniBowHandleTex" OutName="romani_bow_handle" Format="ci8" Width="16" Height="16" Offset="0x7E0" />
+        <Texture Name="gRomaniBowLimbTex" OutName="romani_bow_limb" Format="ci8" Width="8" Height="8" Offset="0x8E0" />
+        
+        <!-- Romani's Wooden Box DisplayList and Textures. She sits on it after being abducted by the aliens. -->
+        <DList Name="gRomaniWoodenBoxDL" Offset="0xA20" />
+        <Texture Name="gRomaniWoodenBoxTex" OutName="romani_wooden_box" Format="rgba16" Width="32" Height="64" Offset="0xAD0" />
+
+        <!-- Romani Animation -->
+        <Animation Name="gRomaniLookAroundAnim" Offset="0x2A8C" /> <!-- Original name is "rm_kyoro" (onomatopoeic for "looking around restlessly") -->
+
+        <!-- Romani Sleeping DisplayList and Textures -->
+        <DList Name="gRomaniSleepingDL" Offset="0x43A0" />
+        <Texture Name="gRomaniSleepingTLUT" OutName="romani_sleeping_tlut" Format="rgba16" Width="16" Height="16" Offset="0x4CF8" />
+        <Texture Name="gRomaniSleepingDressTex" OutName="romani_sleeping_dress" Format="ci8" Width="16" Height="16" Offset="0x4EF8" />
+        <Texture Name="gRomaniSleepingSkinTex" OutName="romani_sleeping_skin" Format="ci8" Width="8" Height="8" Offset="0x4FF8" />
+        <Texture Name="gRomaniSleepingEarTex" OutName="romani_sleeping_ear" Format="ci8" Width="8" Height="8" Offset="0x5038" />
+        <Texture Name="gRomaniSleepingHairTex" OutName="romani_sleeping_hair" Format="ci8" Width="16" Height="16" Offset="0x5078" />
+        <Texture Name="gRomaniSleepingHandTex" OutName="romani_sleeping_hand" Format="ci8" Width="16" Height="16" Offset="0x5178" />
+        <Texture Name="gRomaniSleepingDressPatternAndPillowTex" OutName="romani_sleeping_dress_pattern_and_pillow" Format="ci8" Width="8" Height="16" Offset="0x5278" />
+        <Texture Name="gRomaniSleepingEyeTex" OutName="romani_sleeping_eye" Format="rgba16" Width="32" Height="32" Offset="0x52F8" />
+        <Texture Name="gRomaniSleepingBlanketTex" OutName="romani_sleeping_blanket" Format="rgba16" Width="16" Height="128" Offset="0x5AF8" />
+        <Texture Name="gRomaniSleepingMouthTex" OutName="romani_sleeping_mouth" Format="rgba16" Width="32" Height="16" Offset="0x6AF8" />
+
+        <!-- Romani Animations -->
+        <Animation Name="gRomaniRunAnim" Offset="0x7328" /> <!-- Original anme is "rm_run" -->
```

Comment:
Most of the animations *aren't* written as gerund; this one just happens to be right next to animations like `gRomaniSittingAnim`. This is because I thought `gRomaniSitAnim` kind of looked weird. Would you be okay with me changing the following three animations to:
`gRomaniSitAnim`
`gRomaniSitHeadShakeAnim`
`gRomaniSitLookDownAnim`

Or would you prefer me make everything written as gerund?

Reply:
Ohh, I see.
In that case I think this way is fine, no need to worry